designer/client/outputs/output-edit.tsx

```diff
@@ -90,6 +91,11 @@ class OutputEdit extends Component<Props, State> {
           url: formData.get("webhook-url") as string,
         };
         break;
+      case OutputType.SavePerPage:
+        outputConfiguration = {
+          savePerPageUrl: formData.get("save-per-page-url") as string,
```

Comment:
just to sanity check, if you're hosting multiple forms, itll pull from the right output url per form?

Reply:
Yeah, I added a new output configuration for this so we can tell what the correct web-hook is for save per page and still have the other web-hooks in there as well (ones that may require and complete form for submission)